src/citrine/resources/data_concepts.py

```diff
@@ -388,6 +388,68 @@ def register_all(self, models: List[ResourceType], dry_run=False) -> List[Resour
         )
         return [self.build(obj) for obj in response_data['objects']]
 
+    def update_with_data_validation(self, model: ResourceType,
+                                    dry_run: Optional[bool] = False,
+                                    wait_for_response: Optional[bool] = True,
+                                    timeout: float = 2 * 60,
+                                    polling_delay: float = 1.0) -> Union[None, UUID]:
```

Comment:
`Union[None, UUID]` is the same as `Optional[UUID]`, right?

Reply:
Oh I didn't realize that, I guess that's simpler.

Reply:
Fixed